### ISSUE TITLE
detect() returns empty list if video has no cuts

### ISSUE THREAD

**Issue by entron** [Breakthrough/PySceneDetect#282]
I used the example code 
```
scene_list = detect(path, ContentDetector())
```
for some videos like [this one](https://blob.setinhash.com/0_share/-0b1JACFQI0_000002_000012.mp4), however I got an empty list. Obversly there is one scence in the video and I expect to get at least one element. I don't  know whether it is a bug or due to misuse.

**Comment by Breakthrough** [comment]
To always get at least one scene. you can use a `SceneManager` and set `start_in_scene` to `True` when calling `get_scene_list`:

```python
video = open_video(video_path)
scene_manager = SceneManager()
scene_manager.add_detector(ContentDetector())
scene_manager.detect_scenes(video)
scene_list = scene_manager.get_scene_list(start_in_scene=True)
```

The default behaviour of the API is to not return any scenes if no cuts were detected, in order to be more consistent with `ThresholdDetector`.  For the next release I'll add a `start_in_scene` argument to `detect`, but for now you can use the above snippet.

**Comment by Breakthrough** [comment]
Fixed in v0.6.1 branch, will be included in next release of PySceneDetect. Thanks for the report.
